### DISCUSSION TITLE
esbuild integration?

### DISCUSSION THREAD

**Discussion by finnatsea** [Igorbek/typescript-plugin-styled-components#462] [Q&A] [answered]
Longtime user of this plugin with ts-loader. Our team is switching to esbuild for the insane speedup in build times, 20x-30x speedup on our repo. 

It would be great if this plugin could support esbuild.

**Comment by Igorbek** [comment, marked as answer]
Sorry for the very long delay in the response.
It seems `esbuild` doesn't support TS transformers as they use TS AST: https://github.com/evanw/esbuild/issues/1963
